src/pages/Home/components/Map/components/MarkerHtml/index.tsx

```diff
@@ -1,36 +1,40 @@
-import Default from 'assets/images/search/default-image.png';
 import MarkerLogo from 'assets/images/home/location-marker.png';
 import Selected from 'assets/images/home/selected-marker.png';
+import defaultImage from 'assets/images/home/favicon.png';
 import styles from './MarkerHtml.module.scss';
 
-export function MarkerHtml(photo:string | null, name:string) {
+export function MarkerHtml(photo:string, name:string) {
```

Comment:
이런 값들은 조금 추상적으로 작성해도 문제없을 것 같아요
```suggestion
export function MarkerHtml(img:string, name:string) {
export function MarkerHtml(src:string, name:string) {
```

Reply:
수정했습니다. [6eb2187](https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/107/commits/6eb2187578563785ac0975a002a12ac69e840efc)